guide/additional-info/notation.md

```diff
@@ -55,4 +55,4 @@ The angle brackets `<>` are used for generic types or parameterized types, signi
 
 ![TextChannel#send on the docs](./images/send.png)
 
-In this piece of the docs, you can see three type signatures, `StringResolvable or APIMessage`, `MessageOptions or MessageAdditions`, and `Promise<(Message|Array<Message>)>`. The meaning of the word "or" here is the same as `|`.
+In this piece of the docs, you can see three type signatures, `StringResolvable`, `MessagePayload or MessageOptions`, and `Promise<Message>`. The meaning of the word "or" here is the same as `|`.
```

Comment:
The Promise line shouldn't be changed in this way since it signifies the use of `|` keyword